libs/components/packages/src/schematics/migrations/update-8/update-polyfill/update-polyfill.spec.ts

```diff
@@ -146,6 +147,6 @@ describe('update-polyfill.schematic', () => {
     const architect = angularJson.projects['test-app'].architect;
 
     expect(architect.build.options.polyfills).toEqual('src/polyfills.ts');
-    expect(architect.test.options.polyfills).toEqual('src/test.ts');
```

Comment:
I incorrectly assumed that the `polyfills` config for the test target pointed to `src/test.ts`. Wrong! 😵‍💫